File: logica-programacao/python/036.py
# Peça um número.
# Retorne True se ele não estiver entre 10 e 20 (inclusive).


numero = input("Número: ")
int_numero = int(numero)

verificacao = not (int_numero >= 10 and int_numero <= 20)

string_verificacao = f"O numéro NÃO está entre 10 e 20? {verificacao}"
print(string_verificacao)

File: logica-programacao/javascript/036.js
/* Peça um número.
Retorne True se ele não estiver entre 10 e 20 (inclusive). */ 

let numero = prompt("Número: ");
let intNumero = parseInt(numero);

let verificacao = !(intNumero >= 10 && intNumero <= 20);

let stringVerificacao = `O número NÃO está entre 10 e 20? ${verificacao}`;
console.log(stringVerificacao);
